Blog: UNS & Why You Need Data Modeling (#2819)
FlowFuse/website: src/blog/2024/12/why-uns-need-data-modeling.md

---
title: "UNS & Why You Need Data Modeling"
subtitle: Why data modeling is key to making your Unified Namespace work effectively.
description: Discover why data modeling is crucial for a Unified Namespace (UNS) in manufacturing and how it helps organize and make data actionable.
date: 2024-12-04
authors: ["sumit-shinde"]
image: 
keywords: uns, data modeling, why uns needs data modeling, unified namespace
tags:
   - flowfuse

>    - uns

Reviewer: Hi @sumitshinde-84,  

We now have a blog page dedicated to UNS articles (check https://github.com/FlowFuse/website/pull/2855). To have articles added to this collection, they should include the `unified-namespace` tag.  

You can find the list of tags in the [handbook](https://flowfuse.com/handbook/customer/marketing/blog/#tags).

```suggestion
   - unified-namespace
```

Author: Thanks Yndira, I will update it for all our latest uns blogs

With all these benefits in mind, the next logical question is: How do you implement data modeling effectively, and which platform can help you achieve it? We've only scratched the surface here—data modeling is a much deeper, more dynamic process that can reshape how you use data across your entire business. It’s not just about structuring and integrating data; it’s about transforming your operations, unlocking new efficiencies, and driving growth. The right platform can help you go beyond essential organization and truly harness the power of your data. So, what’s your next step to turn your data into a strategic asset?

## **Leverage FlowFuse for Effective Data Modeling in Your UNS**

[[FlowFuse](/) makes it easy to manage and build your own Unified Namespace (UNS) and improve data operations. It helps connect IT and OT systems and streamline workflows. With FlowFuse, you can quickly connect (integrate different services, hardware, and APIs), collect (aggregate data), transform (transform and contextualize data), and visualize (build dashboards with a low-code approach) data for more efficient processes.

> If you haven’t built your own Unified Namespace (UNS), please read [this article](/blog/2024/11/building-uns-with-flowfuse/) that shows how you can create a UNS in less than 15 minutes with FlowFuse.

Reviewer 2: ```suggestion
If you are looking to build your own Unified Namespace (UNS), see [this article](/blog/2024/11/building-uns-with-flowfuse/) that shows how you can create a UNS in less than 15 minutes with FlowFuse.
```

> FlowFuse is built on Node-RED, a simple platform that lets you create low-code flows to turn raw data from machines and sensors into organized, useful models. It helps structure data, define connections, and add context to keep everything consistent.

Reviewer 2: ```suggestion
FlowFuse leverages Node-RED, the open source low-code platform that helps you create simple flows to turn raw data from machines and sensors into organized, useful models. It helps structure data, define connections, and add context to keep everything consistent.
```

With over 5,000 community nodes and support for protocols like MQTT, OPC-UA, and Modbus, FlowFuse makes it easy to connect different systems and unify your data.

> *Read this [article](/blog/2023/12/unified-namespace-data-modelling/) where Marian shows how to use FlowFuse for data modeling.*

Reviewer 2: ```suggestion
*Read this [article](/blog/2023/12/unified-namespace-data-modelling/) where you can see how to use FlowFuse for data modeling.*
```

> With FlowFuse's enterprise layer, you can manage edge devices and Node-RED instances in one place, making it easier to collaborate, scale, and stay secure as you grow.

Reviewer 2: ```suggestion
With FlowFuse's enterprise-grade features, you can manage edge devices and Node-RED instances in one place, making it easier to collaborate, scale, and stay compliant as you grow.
```

- **Collect**: Aggregate data from machines, sensors, and other sources.  
- **Transform**: Easily standardize and add context to raw data, making it more meaningful and usable across your systems.  
- **Visualize**: Create dashboards with a low-code approach to monitor and analyze your operations.  

Learn how to build your UNS in just 15 minutes with [this article](/blog/2024/11/building-uns-with-flowfuse/).

FlowFuse uses Node-RED, an open-source low-code platform, to turn unstructured data into organized, actionable models. Its ability to transform and contextualize data ensures consistency and clarity, helping you get the most value from your data.

With support for over 5,000 community nodes and protocols like MQTT, OPC-UA, and Modbus, FlowFuse simplifies connecting systems and unifying data.

Check out [this article](/blog/2023/12/unified-namespace-data-modelling/) to see how FlowFuse makes data modeling easier.

FlowFuse also offers enterprise-grade features to manage edge devices and Node-RED instances, helping you scale, collaborate, and stay compliant.

**Take Control of Your Data Operations with FlowFuse**

> Ready to optimize your data operations? Build your Unified Namespace and data model with FlowFuse today. [Get started now](https://app.flowfuse.com/account/create) and unlock the full potential of your data.

Reviewer 3: Can we make this a CTA to discuss how to implement a UNS with our sales team?
